lib/SpamExperts_API.php

```diff
@@ -63,6 +63,12 @@ public function getDomainStatus($domain){
 		if (isset($json['result']['present'])){
 			return (bool)$json['result']['present'];
 		}
+
+		// This is somewhat of an edge case apparently. If it happens the domain isn't there.
+		if (preg_match('/does not belong to the admin/i', implode('. ', $json['messages']['error']))) {
+			return FALSE;
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* TRUE, FALSE and NULL must be lowercase; expected "false" but found "FALSE"

<!-- Comment by [SideCI](https://sideci.com) -->